juniper_subscriptions/Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2018"
 
 
 [dependencies]
-futures = "0.3.1"
+futures = { version = "=0.3.5", features = ["compat"] }
```

Comment:
The same applies here:
- we don't want to have high the most bottom version, unless we really need it in out code;
- nor we want to pin `futures` crate to the exact version `0.3.5`, so introduce any problems with transitive dependencies resolving for library users.